docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -3110,6 +3120,58 @@ definitions:
         example: 'data.security.team'
 
 
+  Annotations:
+    type: object
+    description: >
+      # [Work in progress] Annotations of the Nakadi resource.
+
+      A Nakadi event-type or a subscription is considered as a Nakadi resource.
+      A Nakadi resource can have a set of annotations associated with it.
+      They follow the same rules of annotations in Kubernetes resources.
+        For keys:
+      > Annotations are key/value pairs. Valid annotation keys have two segments:
+      an optional prefix and name, separated by a slash (`/`). The name segment is
```

Comment:
I understand we want to impose as many restrictions as possible because it is easier to relax them than adding them later, but at the same time I am not sure we need be that strict about format for the key: `Valid annotation keys have two segments: an optional prefix and name, separated by a slash (`/`).` The slash could be a part of the allowed symbols, that simplifies annotations / labels definition, but at the same type flexible enough and restrictive

Reply:
aligned with Thomas offline. the K8s rules to define annotation / labels format is optional, I misread. 
it would be nice to clear separate that Nakadi would like to have annotations / labels defined as K8s formatted annotations / labels with prefixe, name and backslash and the strict requirement on the allowed symbols and length.